Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.m

```diff
@@ -96,6 +99,21 @@ + (instancetype)vok_addWithDictionary:(NSDictionary *)inputDict forManagedObject
     }
 }
 
++ (void)vok_addWithArrayInBackground:(NSArray *)inputArray completion:(VOKManagedObjectReturnBlock)completion
+{
+    [VOKCoreDataManager importArrayInBackground:inputArray
+                                       forClass:[self class]
+                                     completion:^(NSArray *arrayOfManagedObjectIDs) {
+                                         NSMutableArray *returnArray = [NSMutableArray array];
+                                         NSManagedObjectContext *moc = [[VOKCoreDataManager sharedInstance] managedObjectContext];
+                                         for (NSManagedObjectID *objectID in arrayOfManagedObjectIDs) {
+                                             [returnArray addObject:[moc objectWithID:objectID]];
+                                         }
+                                         completion([returnArray copy]);
```

Comment:
Need to check if `completion` is set before executing it. That should probably be checked at the top of this completion block, since the objects don't need to be loaded from Core Data if there's no completion block to pass them to.